UIKitDZ/UIKitDZ/View Controllers/SignInViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ViewController.swift
+//  UIKitDZ
+//
+//  Created by Анастасия Козлова on 21.09.2022.
+//
+
+import UIKit
+/// SignInViewController
+class SignInViewController: UIViewController {
+    
+    var storage = Storage()
```

Comment:
```suggestion
    var storage = Storage()
```
обычный вар ниже view или аутлета